Xamarin.PropertyEditing.Mac/Controls/BasePointEditorControl.cs

```diff
@@ -42,10 +48,19 @@ protected BasePointEditorControl (IHostResourceProvider hostResources)
 			AddSubview (YEditor);
 
 			this.AddConstraints (new[] {
-				NSLayoutConstraint.Create (XEditor, NSLayoutAttribute.Width, NSLayoutRelation.Equal, 1f, 90f),
+				NSLayoutConstraint.Create (XLabel, NSLayoutAttribute.Top, NSLayoutRelation.Equal, this,  NSLayoutAttribute.Top, 1f, 16f),
```

Comment:
Shouldn't the labels constrain relative to what they're labeling? 

`NSLayoutConstraint.Create (XLabel, NSLayoutAttribute.Top, NSLayoutRelation.Equal, XEditor,  NSLayoutAttribute.Bottom, 1f, 2),`

Reply:
Also why does this happen here when the other label constraints are added in the base?

Reply:
IIRC this caused a constraints crash/collision, but I'll double check it.

Reply:
Fixed.